openreview/venue/process/simple_paper_recruitment_process.py

```diff
@@ -0,0 +1,150 @@
+def process(client, note, invitation):
+    from Crypto.Hash import HMAC, SHA256
+    import urllib.parse
+    from datetime import datetime
+    VENUE_ID = ''
```

Comment:
do we need this file? can we read the variable from the domain group?

Reply:
yes, we can, I haven't gotten around to updating this yet